pkg/flow/grpc/internal.proto

```diff
@@ -13,6 +13,7 @@ message ReportActionResultsRequest {
 	string errorCode = 4;
 	string errorMessage = 5;
 	bytes output = 6;
+	int32 Iterator = 7;
```

Comment:
Inconsistent case with the rest of the struct fields. Should be named `iterator`. (lowercase 'i')